### PR TITLE
Improvement/update for cool multiaccount

### PR DESCRIPTION
**Pull request by jsf9k**
## 🗣 Description

In this pull request I update this repo to work with our current implementation of the COOL multi-account environment.

## 💭 Motivation and Context

This code is to be deployed to the COOL.  Therefore it must work with our COOL multi-account environment.

## 🧪 Testing

I deployed these changes to production, and I verified that all the pre-commit hooks passed.

## 🚥 Types of Changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [x] Breaking change (causes existing functionality to change)

## ✅ Checklist

- [x] My code follows the code style of this project.
- [x] My change requires a change to the documentation.
- [x] I have updated the documentation accordingly.
- [x] I have read the **CONTRIBUTING** document.
- [ ] I have added tests to cover my changes.
- [x] All new and existing tests passed.

### CODE REVIEW

File: provisionopenvpn_policy.tf [hunk 1]
```diff
@@ -0,0 +1,23 @@
+# ------------------------------------------------------------------------------
+# Create the IAM policy that allows all of the permissions necessary
+# to provision OpenVPN in the Shared Services account.
+# ------------------------------------------------------------------------------
+
+data "aws_iam_policy_document" "provisionopenvpn_policy_doc" {
+  # This policy needs to be more restricted, obviously
```
Comment (dav3r) [inline provisionopenvpn_policy.tf:7]: 🎗Reminder to create an issue to go back and restrict this policy (if this PR is merged and this policy is still wide open).

**Review by dav3r — APPROVED** [review]
So far, so good! 👍 
I left a familiar-looking reminder in my one comment...

**Review by felddy — APPROVED** [review]
Looks good sir.  I'm getting the tingles. 🎇

File: openvpn.tf [hunk 2]
```diff
@@ -30,16 +30,16 @@ module "openvpn" {
   freeipa_admin_pw        = var.freeipa_admin_pw
   freeipa_realm           = upper(var.cool_domain)
   hostname                = "vpn.${var.cool_domain}"
-  private_networks        = var.private_networks
-  private_reverse_zone_id = var.private_reverse_zone_id
-  private_zone_id         = var.private_zone_id
+  private_networks        = [data.terraform_remote_state.networking.outputs.vpc.cidr_block]
+  private_reverse_zone_id = data.terraform_remote_state.networking.outputs.public_subnet_private_reverse_zones["10.128.9.0/24"].id
```
Comment (dav3r) [inline openvpn.tf]: Can `"10.128.9.0/24"` be replaced with a variable here and below?

Comment (jsf9k) [reply openvpn.tf]: See commit 5996486.

Comment (dav3r) [reply openvpn.tf]: Gravy! 👍

**Pull request merged**